### DISCUSSION TITLE
How to use something like CSS classes?

### DISCUSSION THREAD

**Discussion by Zocker1999NET** [Alexays/Waybar#2909] [Q&A]
I want to style the timezone hint different from the normal clock time. I tried following:
```json
    "clock": {
        "format": "{0:%H:%M} <span class='timezone'>{0:%Z}</span>",
    },
```
with following in the stylesheet:
```css
#clock .timezone {
    font-size: 60%;
}
```

However, with this, waybar 9.24 fails to display the clock and logs:
```
Failed to set text '<markup>15:29 <span class="timezone">UTC</span></markup>' from markup due to error parsing markup: Attribute 'class' is not allowed on the <span> tag on line 1 char 46
```

So, as I understand it, I cannot use CSS classes just like in HTML. Is there a similar concept available so I can achieve the different format?

**Comment by jramrath** [comment]
[This](https://docs.gtk.org/Pango/pango_markup.html) is some documentation for the markup waybar uses.

I think what you need is `<span size='60%'>`
